docs/userGuide/markBindSyntax.md

```diff
@@ -248,6 +248,19 @@ MarkBind adds several Markdown-like features on top of GFMD.
 - (X) Option 2 (selected)
 </div>
 
+#### Inline code / syntax highlighting
+
+<div class="indented">
+
+{{ icon_example }} You can syntax highlight inline code by using `{.language}`:
+```markdown
+Consider the Python code `print('Hello', name)`{.python}.
+```
+{{ icon_arrow_down }}
+
+Consider the Python code `print('Hello', name)`{.python}.
```

Comment:
I would like to showcase the syntax highlighting a bit more. Maybe have something like this?

```markdown
Consider the Java method signature: `public static void main(String[] args)`{.java}.
```

Reply:
Got it!